guide/process/README.md

```diff
@@ -0,0 +1,40 @@
+# Orchestration system
+
+The orchestration system is an event-driven task-orchestration system that managed the connection between events and tasks of services by following the steps described by processes.
+
+It lets you build and run complex applications by describing their business logic and let the Engine executes them for you.
+
+Processes are an evolution of [Applications](../application/) as they are managed by the Engine and will be distributed on the upcoming decentralized network.
+
+
+__TODO: rename App by Process on the following image:__
+
+![Application](/application.svg)
+
+## What is a process
+
+A [process](process-file.md) is a list of steps that describes a specific business logic.
+
+Step can be one of the following:
+- Event trigger
+- Task's result trigger
+- Execute a task
+- Condition on the data
+
+Only the first step of a process is and must be an event or a task's result trigger.
+
+### State machine
+
+Processes are [nondeterministic finite](https://en.wikipedia.org/wiki/Nondeterministic_finite_automaton) [state machines](https://en.wikipedia.org/wiki/Finite-state_machine) that perform state transition when matching events or task's results occur but don't support cycle.
```

Comment:
Having this in a specific section is a bit weird.

Let's move the content of state machine and activity diagram to the `what is a process`